dockstore-integration-testing/src/test/java/io/dockstore/common/LocalStackTestUtilities.java

```diff
@@ -32,7 +32,7 @@
 
 public final class LocalStackTestUtilities {
     public static final String IMAGE_TAG = "1.3.1";
-    public static final String ENDPOINT_OVERRIDE = "http://localhost:4566";
+    public static final String ENDPOINT_OVERRIDE = "https://s3.localhost.localstack.cloud:4566";
```

Comment:
Updating the AWS SDK broke this. https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/examples-s3.html

> From version 2.18.x and onward, the AWS SDK for Java 2.x uses [virtual hosted-style addressing](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#virtual-hosted-style-access) when including an endpoint override.

I updated it to this endpoint that localstack recommends https://docs.localstack.cloud/user-guide/aws/s3/#path-style-and-virtual-hosted-style-requests